client/components/Quiz/QuizList.vue

```diff
@@ -0,0 +1,36 @@
+<template>
+  <div class="h-full w-full">
+    <div class="grid grid-rows-auto mr-10 flex space-y-2">
```

Comment:
There is `grid` and `flex` in the same element..... 
Might wanna check that because `grid` and `flex` are two different display types.